### PR TITLE
Add BioPandas to conda-forge

### PR DESCRIPTION
**Pull request by souravsingh**

### CODE REVIEW

File: recipes/biopandas/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+test:
+  imports:
+    - biopandas
+    - biopandas.pdb
+    - biopandas.testutils
+
+about:
+  home: http://github.com/rasbt/biopandas
+  license: BSD
```
Comment (ocefpaf) [inline recipes/biopandas/meta.yaml]: Do you know if the license is `BSD 2-Clause` or `BSD 3-Clause`?

File: recipes/biopandas/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  license_file: LICENSE
+  summary: 'Molecular Structures in Pandas DataFrames'
+
+  # The remaining entries in this section are optional, but recommended
```
Comment (ocefpaf) [inline recipes/biopandas/meta.yaml]: Please remove all the comments from the example.

File: recipes/biopandas/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
+    - pandas
```
Comment (ocefpaf) [inline recipes/biopandas/meta.yaml]: Usually, when using `--single-version-externally-managed`, we don't need the run time dependencies to be available at build time (`numpy` and `pandas`) in this case. Can you try removing them? If it does not work just add them back, sometime the authors import them in `setup.py` and they must be present.

**Comment by souravsingh** [comment]
@ocefpaf The recipe is throwing error on py34 build of BioPandas due to missing numpy dependency.

**Comment by ocefpaf** [comment]
@souravsingh something smellw with the `numpy`+`openblas` package in conda-forge. There are many now who are having similar issues. I guess that @conda-forge/numpy and @conda-forge/openblas could chime in to help us here.

**Comment by ocefpaf** [comment]
@souravsingh can you please to copy https://github.com/conda-forge/staged-recipes/pull/2112/commits/0d396ffd2d5231251b6fe0e9106dba19837ecc91 here just to test a theory I have here...

**Comment by ocefpaf** [comment]
@souravsingh I ended up merging those changes in master. So try to rebase instead of adding them yourself. Just to make it clearer for us later on.

**Comment by souravsingh** [comment]
@ocefpaf Sorry I didn't notice the changes. Do I have to revert back and then rebase?

**Comment by ocefpaf** [comment]
> @ocefpaf Sorry I didn't notice the changes.

Not your fault. I was not planning on merging those until I figure out there are many PRs suffering from that issue and that should be in master.

> Do I have to revert back and then rebase?

Do whatever is easier for you. Sometimes a rebase and conflict resolution is easier :smile:

**Comment by souravsingh** [comment]
@ocefpaf Is the recipe ready for merging?

**Comment by ocefpaf** [comment]
I made a few minor comments. Can you take a look at them. Specifically the one about the licenses.

**Comment by souravsingh** [comment]
@ocefpaf I think I have addressed all of them. I looked at the license and it is BSD 3-clause.

**Comment by ocefpaf** [comment]
The mobile interface does not show any changes. weird. I will look again
later today once I get back to my laptop.

**Comment by ocefpaf** [comment]
All good! Thanks @souravsingh!

**Comment by souravsingh** [comment]
@rasbt Pinging you to tell that the conda-forge recipe for biopandas is ready.

**Comment by rasbt** [comment]
Thanks a lot for doing this. I got an email invite about the repo earlier and just checked it out a few hours ago :). To be honest, creating the recipe was somewhere buried on my to-do list, and I am happy to hear that you've already checked it off :). Happy Holidays!

**Pull request merged**